apps/web/src/pages/templates/filter/FilterModal.tsx

```diff
@@ -304,30 +314,38 @@ function EqualityForm({
                   { value: 'SMALLER_EQUAL', label: 'Smaller or equal' },
                   { value: 'IN', label: 'Contains' },
                   { value: 'NOT_IN', label: 'Not contains' },
+                  { value: 'IS_DEFINED', label: 'isDefined' },
                 ]}
                 {...field}
                 data-test-id="filter-operator-dropdown"
+                onChange={(value) => {
+                  field.onChange(value);
+                  value === 'IS_DEFINED' && setValue(`steps.${stepIndex}.filters.0.children.${index}.value`, '');
```

Comment:
```suggestion
              if (value === 'IS_DEFINED') {
             
                 setValue(`steps.${stepIndex}.filters.0.children.${index}.value`, '');
                 }
                 
```

Suggest avoiding this pattern and just using an if statement for better readability